content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -4,96 +4,101 @@ description: Upgrading an ambient mode installation with Helm.
 
-Istiod is the control plane component that manages and configures the proxies to route traffic within an ambient mesh.
+    Because Revisions are intended to be immutable, we recommend choosing a revision name that corresponds with the version of Istio you are installing, such as `1-22-1`. In addition to choosing a new revision name, you should note your current revision name. You can find this by running
 
-{{< text syntax=bash snip_id=upgrade_istiod >}}
-$ helm upgrade istiod istio/istiod -n istio-system
-{{< /text >}}
+    {{< text syntax=bash snip_id=list_revisions >}}
+    $ kubectl get mutatingwebhookconfigurations -l 'istio.io/rev,!istio.io/tag' -L istio\.io/rev
+    $ # Store your revision and new revision in variables:
+    $ export REVISION=istio-1-22-1
+    $ export OLD_REVISION=istio-1-21-2
+    {{< /text >}}
 
-### Upgrade the ztunnel component
+## Install the New Control Plane
 
-The ztunnel DaemonSet is the node proxy component.
+Istiod is the control plane component that manages and configures the proxies to route traffic within an ambient mesh. The following command will install a new instance of this control plane alongside the old, but will not introduce any new proxies, or take over control for existing proxies. If you have previously customized your istiod installation, you can reuse the `values.yaml` file from previous upgrades or installs to keep your control planes consistent.
 
-{{< warning >}}
-As ambient mode is not yet Stable, the following statement is not a compatibility guarantee, and is subject to change, or removal. Prior to reaching Stable status, this component and/or the control plane may receive breaking changes that prevent compatibility between minor versions.
-{{< /warning >}}
+{{< text syntax=bash snip_id=upgrade_istiod >}}
+$ helm install istiod-"$REVISION" istio/istiod -n istio-system --set revision="$REVISION"
+{{< /text >}}
+
+## Upgrade the ztunnel DaemonSet
 
-The ztunnel at version 1.x is generally compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before ztunnel, as long as their version difference is within one minor version.
+The ztunnel DaemonSet is the node proxy component. The ztunnel at version 1.x is compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before ztunnel, as long as their version difference is within one minor version.
 
 {{< warning >}}
 Upgrading ztunnel in-place will briefly disrupt all ambient mesh traffic on the node.
 Node cordoning and blue/green node pools are recommended to mitigate blast radius risk during production upgrades. See your Kubernetes provider documentation for details.
 {{< /warning >}}
 
 {{< text syntax=bash snip_id=upgrade_ztunnel >}}
-$ helm upgrade ztunnel istio/ztunnel -n istio-system
+$ helm upgrade ztunnel istio/ztunnel -n istio-system --set revision="$REVISION"
```

Comment:
nit: do we need to tell them to pass the original values? Is that even needed or does helm automagically do it